cluster/addons/metrics-server/resource-reader.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: system:metrics-server
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "*"
```

Comment:
why does the metrics server require the ability to read every secret in the system?

Reply:
It requires ability to list/watch pods, nodes and namespaces.

Reply:
then enumerate pods, nodes, and namespaces

Reply:
+1

Reply:
Done